src/arrayMethodSort.js

```diff
@@ -5,8 +5,31 @@
  */
 function applyCustomSort() {
   [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+    if (!compareFunction) {
+      for (let i = 0; i < this.length - 1; i++) {
+        for (let j = 0; j < this.length - i - 1; j++) {
```

Comment:
Try to refactor the code to reduce duplication and improve readability. There is a lot of repeatable code for now.